app/user/rest.py

```diff
@@ -434,7 +435,38 @@ def send_already_registered_email(user_id):
     send_notification_to_queue(saved_notification, False, queue=QueueNames.NOTIFY)
 
     return jsonify({}), 204
+ 
 
+@user_blueprint.route('/<uuid:user_id>/new-registration-data-email', methods=['POST'])
+def send_new_registration_data_email(user_id):
+    data, errors = new_registration_user_data_schema.load(request.get_json())
+
+    API_URL = current_app.config['FRESH_DESK_API_URL']
+    API_KEY = current_app.config['FRESH_DESK_API_KEY']
+
+    ticket = {
+        'product_id': 61000000046,
+        'subject': f"{data['email']} just signed up to Notify!",
+        'description': data['message'],
+        'email': f"name: {data['name']}, email: {data['email']}, date: {data['date']}",
+        'priority': 1,
+        'status': 2,
+    }
+
+    response = requests.post(
+        "{}/api/v2/tickets".format(API_URL),
+        json=ticket,
+        auth=HTTPBasicAuth(API_KEY, "x")
+    )
+
+    if response.status_code != 201:
+        print("Failed to create ticket, errors are displayed below")
+        content = json.loads(response.content)
+        print(content["errors"])
+        print("x-request-id : {}".format(content.headers['x-request-id']))
+        print("Status Code : {}".format(str(content.status_code)))
+
+    return jsonify({"status_code": response.status_code}), 204
```

Comment:
This logic is almost the same as in `send_support_email` in the same file. Could you refactor to have a common method for both?